src/blockchain/ain/jellyfish/utxo-provider.service.ts

```diff
@@ -155,6 +158,30 @@ export class UtxoProviderService {
     return neededUnspent;
   }
 
+  private static provideUntilAmountPlusFee(
```

Comment:
Method name doesn't really fit IMO 🤔 perhaps something like `tryProvideUntilAmount`?

Reply:
good suggestion, I was completely blank what I should name that thing